embed/vite.config.ts

```diff
@@ -5,7 +5,21 @@ export default defineConfig(({ mode }) => {
   const isDevelopment = mode === 'development'
 
   return {
-    envDir: resolve(__dirname),
+    define: {
+      'import.meta.env.VITE_SCRIPT_FRONTEND_URL': JSON.stringify(
+        isDevelopment
+          ? 'http://localhost:3000'
+          : 'https://webmonetization.org.pages.dev/tools/'
+      ),
+      'import.meta.env.VITE_SCRIPT_API_URL': JSON.stringify(
+        isDevelopment
+          ? 'http://localhost:8787'
```

Comment:
Similarly for this. We use string concatenation when building URLs, so need to be careful if we want our constants to have trailing `/` or not/
```suggestion
          ? 'http://localhost:8787/'
```